DeepSpeech.ipynb

```diff
@@ -1322,7 +1303,7 @@
     "    tower_total_losses, \\\n",
     "    tower_avg_losses, \\\n",
     "    avg_accuracy \\\n",
-    "    = get_tower_results(data_sets.train, optimizer)\n",
+    "    = get_tower_results([data_sets.train, data_sets.dev, data_sets.test], optimizer)\n",
     "    \n",
     "    # Validation results parameters\n",
     "    dev_results_params = get_results_params(data_sets.dev)\n",
```

Comment:
This line needs to be removed. (I'd guess it's a leftover from the merge.)

The method `get_results_params()` doesn't exist anymore; so if this line isn't removed, the code will fail
```
Traceback (most recent call last):
  File "<stdin>", line 957, in <module>
  File "<stdin>", line 835, in train
NameError: global name 'get_results_params' is not defined
```